Java/RollDice/README.md

```diff
@@ -0,0 +1,21 @@
+# RollDice  
+
+## Introduction
+```
+It is the program that will help you to Roll the Dice.
```

Comment:
Please elaborate the introduction to how this program used ```Random ()``` function to achieve the task.